iati/codelists.py

```diff
@@ -209,8 +215,13 @@ def __eq__(self, other):
         Todo:
             Utilise all attributes as part of the equality process.
 
+            Test comparison with strings.
+
         """
-        return ((self.name) == (other.name)) and ((self.value) == (other.value))
+        if isinstance(other, iati.codelists.Code):
```

Comment:
Consider making this block more pythonic, but optional whether this is worth changing now.

Reply:
https://github.com/IATI/pyIATI/pull/209/commits/3050de8639068bb154d17c77d997d5342ad18e85